lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1027,7 +1027,7 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
+    Generate = [{rsa, 2048, 3}, {rsa, 3072, 65537}],
```

Comment:
Strange...  Unfortunately the test logs has been removed because they are kept only a couple of days.  I've added the branch to testing again so we can see tomorrow what the message was.